tests/e2e/mocha-python-django.opts

```diff
@@ -0,0 +1,8 @@
+--timeout 2200000
```

Comment:
If we agree on adding single mocha*.opts to every test we have, the root of tests would be spammed by mocha files. I don't see this file necessary as tests are used just to run in a whole bunch of tests (e.g. in the nightly job). 
If you want to keep single file for each test, I would go for creating some folder for mocha*.opts files so the root will not grow to enormous number of files. (It doesn't have to be done as a part of this PR, feel free to create an issue for it.)

Reply:
Should we delete the vertx and mavent mocha-opts then as well? Are they used separately anywhere?

Reply:
I don't think they are used anywhere in jobs.

Reply:
So far I've only removed the python mocha. I will probably send a separate PR for removing the vertx one. The maven one is already not present in the repo.